isaric/parsers/ccp-guinea.toml

```diff
@@ -659,128 +636,140 @@
 [visit.treatment_inotropes_vasopressors]
   field = "vaso_or_inot"
   description = "Any administration of vasopressor/inotrope?"
-  ref = "Y/N/NK"
+  ref = "Y/N"
 
 [visit.treatment_invasive_ventilation]
-  field = "overall_invasive_proccur"
-  description = "Invasive ventilation?"
-  ref = "Y/N/NK"
+  field = "inv_vent"
+  ref = "Y/N"
 
 [visit.treatment_noninvasive_ventilation]
-  field = "overall_noninvasive_proccur"
+  field = "non_inv_vent"
   description = "Non-invasive ventilation (eg BiPAP, CPAP)?"
-  ref = "Y/N/NK"
+  ref = "Y/N"
 
 [visit.treatment_other]
   combinedType = "set"
   excludeWhen = "none"
   fields = [
     { field = "other_meds_1", description = "Other treatment" },
     { field = "other_meds_2", description = "Other treatment" },
+    { field = "other_intervs_specify" },
   ]
 
 [visit.treatment_oxygen_therapy]
   field = "overall_oxygen_cmoccur"
   description = "Oxygen therapy?"
-  ref = "Y/N/NK"
+  ref = "Y/N"
```

Comment:
to add other oxygen therapy variables to this attribute (non-inv, invasive etc.), perhaps I'm not seeing those in this amendment